src/app/tecnologia/tecnologia.component.ts

```diff
@@ -0,0 +1,37 @@
+import { BreakpointObserver, Breakpoints } from '@angular/cdk/layout';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/brunodiascultural/bookshelf_v1/pullRequest?prid=8812417)
```suggestion
import { BreakpointObserver, Breakpoints } from "@angular/cdk/layout";
```